local_costmap:
  update_frequency: 5.0
  publish_frequency: 5.0
  transform_tolerance: 0.5
  static_map: false
  rolling_window: true
  width: 9
  height: 9
  origin_x: -1.5
  origin_y: -1.5
  resolution: 0.01
  inflation_radius: 0.25
  #track_unknown_space: true
